pkg/azdiskschedulerextender/azdiskschedulerextender.go

```diff
@@ -156,27 +169,10 @@ func filter(context context.Context, schedulerExtenderArgs schedulerapi.Extender
 		}
 	}
 
-	// get PV names for requested PVCs
-	var pvNamesForRequestedVolumes []string
-	numberOfRequestedAttachments := 0
-	for _, volume := range requestedVolumes {
-		if volume.PersistentVolumeClaim != nil {
-			// not the same as len(pvNamesForRequestedVolumes). PV-s for a new pod might not be cached yet.
-			numberOfRequestedAttachments++
-			if entry, ok := pvcToPvMapCache.Get(volume.PersistentVolumeClaim.ClaimName); ok {
-				klog.V(2).Infof(
-					"pvc(%s) to pv(%s) mapping for pod %s exists.",
-					volume.PersistentVolumeClaim.ClaimName,
-					entry.VolumeName,
-					schedulerExtenderArgs.Pod.Name,
-				)
-				pvNamesForRequestedVolumes = append(pvNamesForRequestedVolumes, entry.VolumeName)
-			}
-		}
-	}
-
-	if numberOfRequestedAttachments == 0 {
```

Comment:
Why remove this line? If the pod only has the default volume, we don't want to continue the scheduling.